editoast/openapi.yaml

```
@@ -572,6 +572,12 @@ paths:
             type: integer
           description: Infra ID
           required: true
+        - in: query
+          name: include_rolling_stocks_modes
```

Inconsistent with the real option name:

```suggestion
          name: include_rolling_stock_modes
```